src/WebJobs.Script/Utility.cs

```diff
@@ -280,5 +281,19 @@ public static JObject ToJObject(ExpandoObject value)
             string json = ToJson(value);
             return JObject.Parse(json);
         }
+
+        public static bool TryMatchAssembly(string assemblyName, Type type, out Assembly matchedAssembly)
```

Comment:
This really belongs in the ScriptBindingProvider base class, but I don't want to do a separate update/release for that to make that happen, primarily because that Extension assembly is going to go away anyways.